accel-pppd/iprange.c

```diff
@@ -220,8 +220,9 @@ int __export iprange_tunnel_check(in_addr_t ipaddr)
 	return res;
 }
 
-static void iprange_load_config(void *data)
+static void iprange_load_config()
```

Comment:
event func accepts argument parameter (although can be unused), this change seems wrong.
as for similar cases, please, either revert or cast to triton_event_func

Reply:
grep "load_config()" -R * |grep -c ''
49
grep "load_config(void" -R * |grep -c ''
35

Now function without parameters is dominating. So we need to replace all such places?